plugins/out_stackdriver/stackdriver.c

```diff
@@ -2343,6 +2343,25 @@ static void update_http_metrics(struct flb_stackdriver *ctx,
         cmt_counter_inc(ctx->cmt_requests_total, ts, 2, (char *[]) {tmp, name});
     }
 }
+
+static void update_retry_metric(struct flb_stackdriver *ctx,
+                                 struct flb_event_chunk *event_chunk,
+                                 uint64_t ts,
+                                 int http_status, int ret_code)
+{
+    if (ret_code != FLB_RETRY) {
+        return;
+    }
+
+    char tmp[32]; 
```

Comment:
define vars at the top of the function: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#variable-definitions

Reply:
Done.